.maintain/runtime_release.sh

```diff
@@ -38,7 +38,7 @@ build_runtime() {
 for i in "${VERSIONS_FILES[@]}"; do
   while IFS=',' read -r output chain folder; do
     echo "check if the wasm sources changed for $chain"
-    if has_runtime_changes "${PREV_TAG}" "${GITHUB_REF_NAME}" "$folder"; then
+    if has_runtime_changes "${PREV_TAG}" "${GITHUB_BRANCH_NAME}" "$folder"; then
```

Comment:
this also works on push not pull_request

Reply:
Updated